metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/common/LambdaExpression.java

```diff
@@ -56,9 +57,9 @@ public Object apply(List<Object> variableArgs) {
       lambdaVariables.put(variables.get(i), null);
     }
 
-    VariableResolver variableResolver = variable -> lambdaVariables.getOrDefault(variable
+    VariableResolver variableResolver = new DefaultVariableResolver(variable -> lambdaVariables.getOrDefault(variable
```

Comment:
The only impact would be in the tests right now.  Since we use the MapResolver in most places ( shell, profiler, enrichments, parser ).